src/pages/whats-new.js

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import SEO from '../components/seo';
+import PageTitle from '../components/PageTitle';
+import { graphql } from 'gatsby';
+import { Layout, Link } from '@newrelic/gatsby-theme-newrelic';
+
+const MOBILE_BREAKPOINT = '960px';
+
+const WhatsNew = ({ data }) => {
+  const { allMarkdownRemark } = data;
+  const posts = allMarkdownRemark.edges.map(({ node }) => node);
+  const postsByDate = Array.from(
+    posts
+      .reduce((map, post) => {
+        const key = post.frontmatter.releaseDate;
+
+        return map.set(key, [...(map.get(key) || []), post]);
+      }, new Map())
+      .entries()
+  );
+
+  return (
+    <>
+      <SEO title="What's new in New Relic" />
+      <div>
+        <PageTitle
+          css={css`
+            margin-bottom: 2rem;
+          `}
+        >
+          What's new in New Relic
```

Comment:
@jpvajda / @austin-schaefer Should we be writing `What's new` or `What's New` (with a capital `N`) in the UI?

Reply:
Should be sentence case in general. Looks like current Drupal site using title case but I'm editing it.

So `What's new` at the start of a sentence, or `what's new` within a sentence.